README.md

```diff
@@ -0,0 +1,79 @@
+# Howl
+
+Because somebody should yell at you when your `master` build breaks.
+
+Howl is a GitHub Action. It writes a custom Slack message with an
+optional `@here` or `@project-owners` to your preferred Slack channel.
+
+<div style="text-align:center;width:100%">
+	<img src="howler.jpeg" alt="Howler Monkey"/>
+	<br/>
+	<sub>Howler Monkay -
+		<a href="https://commons.wikimedia.org/wiki/File:DSC09108_-_Guyanan_Red_Howler_Monkey_(36384553204).jpg">
+			Wikimedia, CC BY-SA 2.0
+		</a>
+	</sub>
+</div>
+
+## Set up Slack token
+
+Set up a repository or organisation secrets with your [Incoming Slack
+Webhook] token. Use, for instance, `SLACK_TOKEN`. The content should
+look similar to
+
+	T01HYRP1APN/B0259MEMKDN/GcXTiEuxU0ZjsGmSPv2kVfmw
+
+Taken from the Slack provided Incoming Webhook URL:
+
+	https://hooks.slack.com/services/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
+
+[Incoming Slack
```

Comment:
Join this line with the next. I don't wrap these - I think it looks funny.

Reply:
yes!